build.gradle

```diff
@@ -4,12 +4,12 @@ plugins {
 }
 
 ext {
-  openTelemetryVersion = "1.14.0"
-  openTelemetryAlphaVersion = "1.14.0-alpha"
+  openTelemetryVersion = "1.15.0"
+  openTelemetryAlphaVersion = "1.15.0-alpha"
   openTelemetryInstrumentationVersion = "1.14.0"
```

Comment:
Should we wait to merge this until 1.15.0 of instrumentation is published as well?

Reply:
I was wondering same with the contrib update (static-instrumenter has a dependency now on instrumentation-api), I'm ok with either waiting or not.

Once things stabilize (and it's purely a version bumping exercise) we should be able to automate.